model/postgres/postgres_resource.rb

```diff
@@ -43,7 +43,8 @@ def hyper_tag_name(project)
   end
 
   def display_state
-    return "running" if ["wait", "refresh_certificates"].include?(strand.label)
+    return "unavailable" if representative_server&.strand&.label == "unavailable"
+    return "running" if ["wait", "refresh_certificates"].include?(strand.label) || strand.label == "refresh_dns_record" && !initial_provisioning_set?
```

Comment:
nit, feel free to ignore:
```suggestion
    return "running" if ["wait", "refresh_certificates", "refresh_dns_record"].include?(strand.label) && !initial_provisioning_set?
```